dts/bindings/pinctrl/cypress,psoc6-pinctrl.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2020, Linaro Limited
+# Copyright (c) 2020, ATL Electronics
+# SPDX-License-Identifier: Apache-2.0
+
+description: Cypress PSoC-6 Pinctrl container node
+
+compatible: "cypress,psoc6-pinctrl"
+
+include: [base.yaml, pincfg-node.yaml]
+
+properties:
+    "#address-cells":
+      required: true
+      const: 1
+    "#size-cells":
+      required: true
+      const: 1
+
+child-binding:
+    description: cypress pins
+    properties:
+      "cypress,pins":
+        type: phandle-array
```

Comment:
Can you provide a usage example of the pinctrl bindings (board)?

Reply:
Yes, to have something that works need solve dependencies first: dt-util.h, IRQ and GPIO.